### PR TITLE
CMS-322 updates tags view

### PR DESCRIPTION
**Pull request by lory87**
**This pull request fixes the following issues**:

- https://divio-ch.atlassian.net/browse/CMS-322

screen shot 2015-09-24 at 15 08 18
screen shot 2015-09-24 at 15 08 24

### CODE REVIEW

**Comment by vxsx** [comment]
the lines should not go into the padding area

File: djangocms_admin_style/sass/components/_content.scss [hunk 1]
```diff
@@ -166,15 +166,42 @@ body {
...
+        margin: 5px -10px 0;
+        border-top: solid 1px $gray-lighter !important;
+        border-bottom: solid 1px $gray-lighter !important;
+    }
+    .add-row {
+        margin-top: 10px;
+    }
+    .inline-deletelink {
+        color: $gray-darkest !important;
+        font: 0/0 a;
```
Comment (dhorelik) [inline djangocms_admin_style/sass/components/_content.scss]: a typo?

**Comment by lory87** [comment]
So it's like matthias designed it.

File: djangocms_admin_style/sass/components/_content.scss (hunk 1 above)
Comment (lory87) [inline djangocms_admin_style/sass/components/_content.scss]: font:0/0 a – a shorthand property that zeros out the font size and line-height. The a value acts as a very short font-family. It's used in bootstrap for hiding text as well

Comment (dhorelik) [inline djangocms_admin_style/sass/components/_content.scss]: cool, didn't know about the `a` value. thanks!

Comment (vxsx) [inline djangocms_admin_style/sass/components/_content.scss]: nope :)

**Comment by vxsx** [comment]
well them i'm clearly wrong :D
(although it does look weird)

**Comment by vxsx** [comment]
LGTM

**Pull request merged**